src/js/extensions/infragistics.ui.combo.knockout-extensions.js

```diff
@@ -55,6 +55,43 @@
         }
     }
 
+    function applyListItemsBindings(valueAccessor) {
+        var combo = valueAccessor().combo,
+            $comboList = combo.igCombo("listItems"),
+            options = valueAccessor().options,
+            dataSource = ko.utils.unwrapObservable(valueAccessor().dataSource),
+            i;
+
+        for (i = 0; i < $comboList.length; i++) {
+            if (ko.isObservable($comboList[ i ])) {
+                ko.applyBindingsToNode($comboList[ i ], {
+                    igComboItem: {
+                        combo: combo,
+                        value: dataSource[ i ],
+                        index: i,
+                        options: options
+                    }
+                }, dataSource[ i ]);
+            }
+        }
+    }
+
+    function identicalDataSources(combo, viewModelDataSource) {
+        var valueKey = combo.igCombo("option", "valueKey"), comboDataSource, index;
+
```

Comment:
If the method gets reused somewhere else it's better to have check for the length of the both dataSources

Reply:
Done